pkg/config/init.go

```diff
@@ -75,6 +75,17 @@ func LoadWithDefault() Conf {
 		c.ModeWebhookType = mwt
 	}
 
+	c.ProvideDockerHubMetrics = false
+	if os.Getenv(EnvProvideDockerHubMetrics) == "true" {
+		c.ProvideDockerHubMetrics = true
+	}
+
+	c.DockerHubCredential = DockerHubCredential{}
+	if os.Getenv(EnvDockerHubUsername) != "" && os.Getenv(EnvDockerHubPassword) != "" {
+		c.DockerHubCredential.Username = os.Getenv(EnvDockerHubUsername)
+		c.DockerHubCredential.Password = os.Getenv(EnvDockerHubPassword)
+	}
```

Comment:
Please logging and handling

- if set EnvProvideDockerHubMetrics is false, but set Docker credential (maybe not needed)
- if set EnvProvideDockerHubMetrics is true, but not set Docker credential (logging anonymous mode)

Reply:
thanks, reflected your reviews.